### PR TITLE
Implement Wiki Update action

### PR DESCRIPTION
**Pull request by Andre601**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [ ] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [x] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [ ] **Other:** `New Docs folder` 

## Description

Implements a system that allows to update the wiki through Pull requests made towards the files in the docs folder.
This has the benefit that we can allow people to contribute towards the wiki, without risking to allow anyone to edit any pages without security measures in place.

Closes #

[contributing]: https://github.com/Readme-Workflows/recent-activity/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

**Review by abhijoshi2k — APPROVED** [review]

**Comment by PuneetGopinath** [comment]
Changelog?

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]
Please update the Changelog

File: docs/README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Docs - Recent Activity
+# Recent Activity Wiki
```
Comment (PuneetGopinath) [inline docs/README.md:1]:
```suggestion
# Welcome Recent Activity's Wiki
```

File: docs/Home.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+Welcome to the recent-activity wiki!
+
+## Pages
+
+- [[Examples]] (Outdated. Use the Configuration page instead!)
```
Comment (PuneetGopinath) [inline docs/Home.md:5]:
```suggestion
- [[Examples]] (Outdated. v1.3.0 and older)
```

**Review by PuneetGopinath — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (Andre601) [reply docs/README.md:1]: Sounds a bit bad imo.

Maybe `Welcome to the Wiki of Recent Activity`?

Comment (PuneetGopinath) [reply docs/README.md:1]: I have updated the suggestion

**Comment by PuneetGopinath** [comment]
> Please update the Changelog

@Andre601

**Comment by Andre601** [comment]
> > Please update the Changelog
> 
> @Andre601

Done 👍

File: docs/README.md (hunk 1 above)
Comment (PuneetGopinath) [reply docs/README.md:1]: @Andre601

**Comment by PuneetGopinath** [comment]
> > > Please update the Changelog
> > 
> > 
> > @Andre601
> 
> Done 👍

Can you also add that we added `docs` folder in #64 in Changelog, I think it will be useful, what's your opinion @abhijoshi2k?

**Review by PuneetGopinath — APPROVED** [review]

**Pull request merged**